src/Congo.WebApi/Startup.cs

```diff
@@ -80,10 +90,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseAuthorization();
 
-            app.UseEndpoints(endpoints =>
-            {
-                endpoints.MapControllers();
-            });
+            app.UseEndpoints(endpoints => { endpoints.MapControllers(); });
```

Comment:
Personal preference: Don't lambda here in case we want to add more endpoints f.e a  SignalR hub or fallback Controller

Reply:
I think what Chris meant here is keep the braces (and on their own lines, as in the old code), as this is something we're likely to add to in the future.